draft-ietf-tls-tls13.md

```diff
@@ -2977,7 +2974,14 @@ Structure of this message:
        opaque ASN1Cert<1..2^24-1>;
 
        struct {
-           ASN1Cert cert_data;
+           select(certificate_type){
+               case RawPublicKey:
+                 // From RFC 7250 ASN.1_subjectPublicKeyInfo
+                 opaque ASN1_subjectPublicKeyInfo<1..2^24-1>;
```

Comment:
Does this really need to be 2^24-1 long?  I assume that we could do with 2^16-1 quite happily.